test/fixtures/common-debuglog.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const common = require('../common');
+
+// argv[0] is the execPath, argv[1] is the test script filename
+common.debuglog.apply(null, process.argv.slice(2));
```

Comment:
Could this be simplified to
```js
common.debuglog(process.argv.slice(2));
```
?

Reply:
that ends up printing `TEST 2461: [ 'message' ]` instead of `TEST 2461: message`, since slice returns an array. So resorted to `common.debuglog(...process.argv.slice(2))` instead.